modules/environment-schema.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "conda environment yaml",
+    "description": "Validate the environment.yml file for an nf-core module",
+    "type": "object",
+    "properties": {
+        "name": {
```

Comment:
We don't add the name now, linting will fail, should we add it to all modules with this PR?

Reply:
ah, yes. let's wait for @Emiller88's PR to add the names first then